src/resources/postcss/tribe-events-single-blocks.pcss

```diff
@@ -118,15 +101,59 @@
 	}
 }
 
-/* Theme specific overrides */
+/* = Organizer block view styles
+============================================= */
+.tribe-block__organizer__details {
+	border-top: 1px solid var(--tec-color-border-default);
+	padding: 22.5px 0;
+
+	@media screen and (min-width: 768px) {
+		padding: 32px 0;
+	}
+
+	h3 {
+		font-size: 1rem;
+		font-weight: normal;
+		letter-spacing: normal;
+		line-height: 1.64;
+
+		a {
+			color: var(--tec-color-link-accent);
+		}
+	}
+
+	p {
+		color: #141827;
+		font-size: 1rem;
+		letter-spacing: normal;
+		line-height: 1.64;
+	}
+}
+
+/* = Theme specific overrides
+============================================= */
 
 /* Twenty-twenty-one theme */
-.tribe-theme-twentytwentyone .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__website a {
-	color: var(--tec-color-link-accent);
+.tribe-theme-twentytwentyone {
+	/* Website block view styles */
+	.tribe-block__event-website a:focus:not(.wp-block-button__link):not(.wp-block-file__button) {
+		background-color: var(--tec-color-link-accent);
+	}
+
+	/* Venue block view styles */
+	.tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__website a {
+		color: var(--tec-color-link-accent);
+	}
 }
 
 /* Twenty-twenty theme */
 .tribe-theme-twentytwenty {
+	/* Website block view styles */
+	.tribe-block__event-website a {
+		font-size: 1.65rem;
+	}
+
+	/* Venue block view styles */
 	.tribe-block__venue {
 		.tribe-block__venue__meta {
```

Comment:
combine these two into one line, no needed to nest unnecessarily

Reply:
done.